Close #35

eswan18 opened the pull request:
This PR adjusts the schedule to close out #35. However, I think a little more schedule updating is required because – based on feedback – we should cut or trim many of the sections on Day 3. If we do this, I think we can move some of the modeling section to Day 3.

bradleyboehmke reviewed (approved):
Did the change to 3.9.2 cause the validation notebook to fail?  Just noticed that the checks didn't pass.

eswan18 commented:
I've added an explicit dependency on the missing package (which was causing the CI to fail), and along with that I updated all the major dependencies. The CI passes so the notebooks should all still run fine.
